mobile_verifier/src/reward_shares.rs

```diff
@@ -924,11 +992,17 @@ mod test {
         // get the summed rewards allocated to subscribers for discovery location
         let mut allocated_mapper_rewards = 0_u64;
         for (reward_amount, subscriber_share) in
-            mapping_shares.into_subscriber_rewards(&epoch, rewards_per_share)
+            shares.into_subscriber_rewards(&epoch, rewards_per_share)
         {
             if let Some(MobileReward::SubscriberReward(r)) = subscriber_share.reward {
-                assert_eq!(expected_reward_per_subscriber, r.discovery_location_amount);
-                assert_eq!(reward_amount, r.discovery_location_amount);
+                assert_eq!(
```

Comment:
can we extend this test so that it also asserts on the individual discovery_location_amount and the verification_mapping_amount along with the existing reward totals